news/4.34/platform.html

```diff
@@ -181,6 +181,29 @@ <h2>General Updates </h2>
         </p>
       </td>
     </tr>
+    
+    <tr id="edge-browser-improvements"> <!-- https://github.com/eclipse-platform/eclipse.platform.swt/issues/1466 -->
+      <td class="title">Several Improvements made to the Edge Browser (Windows only)</td>
```

Comment:
Maybe have a shorter title?
```suggestion
      <td class="title">Improvements to Edge Browser (Windows only)</td>
```

Reply:
Yes, and title case is better.

Reply:
Done